frontend/src/components/Pages/InspectionPage/ScheduleMissionDialog.tsx

```diff
@@ -0,0 +1,159 @@
+import { Autocomplete, Button, Card, Dialog, Typography, Popover } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { useRef, useState, useEffect } from 'react'
+import { useInstallationContext } from 'components/Contexts/InstallationContext'
+import { Robot } from 'models/Robot'
+import { CondensedMissionDefinition } from 'models/MissionDefinition'
+import { BackendAPICaller } from 'api/ApiCaller'
+
+interface IProps {
+    missions: CondensedMissionDefinition[]
+    refreshInterval: number
+    closeDialog: () => void
+}
+
+const StyledMissionDialog = styled.div`
+    display: flex;
+    justify-content: space-between;
+`
+const StyledAutoComplete = styled(Card)`
+    display: flex;
+    justify-content: center;
+    padding: 8px;
+    gap: 25px;
+`
+const StyledMissionSection = styled.div`
+    display: flex;
+    margin-left: auto;
+    margin-right: 0;
+    gap: 10px;
+`
+const StyledDialog = styled(Dialog)`
+    display: flex;
+    padding: 1rem;
+    width: 320px;
+`
+
+export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
+    const { TranslateText } = useLanguageContext()
+    const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false)
+    const [selectedRobot, setSelectedRobot] = useState<Robot>()
+    const [robotOptions, setRobotOptions] = useState<Robot[]>([])
+    const anchorRef = useRef<HTMLButtonElement>(null)
+    const { installationCode } = useInstallationContext()
+
+    useEffect(() => {
+        const id = setInterval(() => {
+            BackendAPICaller.getEnabledRobots().then((robots) => {
+                setRobotOptions(robots)
+            })
```

Comment:
Instead of making the API call here, use the props that are used for echoMissions on MissionOverview/ScheduleMissionDialog so that the autocomplete is faster

Reply:
If you change the refresh interval on this section to refreshInterval instead of slowRefreshInterval, this is no longer necessary